src/App.css

```diff
@@ -1,4 +1,276 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto+Slab:wght@300;500;700&display=swap');
+
 * {
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --white: #fff;
+  --blue: #0290ff;
+  --body: #fafafa;
+  --border: #e8e8e8;
+  --purple: #307bbe;
+  --light-purple: #4386bf;
+}
+
+body {
+  background-color: var(--body);
+}
+
+.navbar {
+  padding: 24px 4%;
+  background-color: var(--white);
+  border-bottom: 1px solid var(--border);
+  display: flex;
+  align-items: center;
+  grid-gap: 40px;
+  gap: 40px;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
thanks for the note